srcpkgs/python3-pathspec/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-pathspec'
 pkgname=python3-pathspec
-version=0.8.0
+version=0.9.0
 revision=1
 wrksrc="pathspec-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3"
```

Comment:
Why? It should still depend on python3, right?

Reply:
> https://github.com/void-linux/void-packages/pull/32746#issuecomment-908549747
> * no need do depend on interpreter when depending on module

Because of this. Or am I misunderstanding?

Reply:
"when depending on module" is key part. If you leave depends empty, no other module will pull in interpreter.

Reply:
> "when depending on module" is key part. If you leave depends empty, no other module will pull in interpreter.

I understood. I thought the word "module" refers to `build_style=python3-module` ;)
Thanks.